CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.1] - 2021-04-27
+### Added
+- Add `TezosSignature2021`/`TezosMethod2021`.
+- Parse Tezos-base58 keys.
+### Changed
+- Use `k256` instead of `libsecp256k1`.
+- Update `ssi-contexts` to `v0.1.0`
+### Fixed
+- Fixed some links.
```

Comment:
Maybe say we fixed an issue with `simple_asn` and `big_int` in case someone who has issues wants to know if it will solve things?

Reply:
@sbihel good idea - after all, that is a main reason for this release.
How about:
```md
### Fixed                                                                       
- Fix dependency conflict between `simple_asn1` and `num-bigint`.
- Fix broken links.
```
^ Also changing to use present tense to match the rest of the changelog items (except for headings which come from the template).
Broken links is about 33720404acdf23501fa6935896177c7f573580ca and 2e7713580fbaa1af553c1b3b1cc4f63495ffdacf.

Reply:
Looks good :+1: